pubsub/api/Program.cs

```diff
@@ -0,0 +1,484 @@
+﻿using CommandLine;
+using Google.Api.Gax.Grpc;
+using Google.Cloud.Iam.V1;
+using Google.Cloud.PubSub.V1;
+using Google.Protobuf;
+using Grpc.Core;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Diagnostics;
+
+namespace GoogleCloudSamples
+{
+    [Verb("createTopic", HelpText = "Create a pubsub topic in this project.")]

```

Comment:
To camelCase or to hyphenate?  CamelCase wins.  Quote me on that if you see me writing commands with hyphenated names.
```
rennie@rennie:~/gitrepos/dotnet-docs-samples$ find . -name "*.cs" | xargs grep -nH -e Verb
./monitoring/api/Program.cs:33:    [Verb("create", HelpText = "Create a metric descriptor for this project.")]
./monitoring/api/Program.cs:43:    [Verb("list", HelpText = "List metric descriptors for this project.")]
./monitoring/api/Program.cs:46:    [Verb("listGroups", HelpText = "List metric groups for this project.")]
./monitoring/api/Program.cs:49:    [Verb("listResources", HelpText = "List monitored resources for this project.")]
./monitoring/api/Program.cs:58:    [Verb("get", HelpText = "Get a metric descriptor's details.")]
./monitoring/api/Program.cs:67:    [Verb("getResource", HelpText = "Get a monitored resource descriptor.")]
./monitoring/api/Program.cs:76:    [Verb("write", HelpText = "Write TimeSeries data to custom metric.")]
./monitoring/api/Program.cs:83:    [Verb("read", HelpText = "Read TimeSeries data from a metric.")]
./monitoring/api/Program.cs:93:    [Verb("readFields", HelpText = "Reads headers of time series data.")]
./monitoring/api/Program.cs:102:    [Verb("readAggregate", HelpText = "Aggregates time series data that matches the specified metric type.")]
./monitoring/api/Program.cs:111:    [Verb("readReduce", HelpText = "Reduces time series data that matches the specified metric type.")]
./monitoring/api/Program.cs:122:    [Verb("delete", HelpText = "Delete a metric descriptor for this project.")]
./language/api/v1/Analyze/AnalyzeTest/AnalyzeTest.cs:92:            Assert.Contains("Verb Conquers", output);
./language/api/v1Beta2/Analyze/AnalyzeTest/AnalyzeTest.cs:92:            Assert.Contains("Verb Conquers", output);
./translate/api/Translate/Translate.cs:22:    [Verb("translate", HelpText = "Translate text.")]
./translate/api/Translate/Translate.cs:39:    [Verb("list", HelpText = "List available languages.")]
./translate/api/Translate/Translate.cs:46:    [Verb("detect", HelpText = "Detects which language some text is written in.")]
./speech/api/Recognize/Recognize.cs:46:    [Verb("sync", HelpText = "Detects speech in an audio file.")]
./speech/api/Recognize/Recognize.cs:49:    [Verb("with-context", HelpText = "Detects speech in an audio file."
./speech/api/Recognize/Recognize.cs:53:    [Verb("async", HelpText = "Creates a job to detect speech in an audio "
./speech/api/Recognize/Recognize.cs:57:    [Verb("sync-creds", HelpText = "Detects speech in an audio file.")]
./speech/api/Recognize/Recognize.cs:68:    [Verb("stream", HelpText = "Detects speech in an audio file by streaming "
./speech/api/Recognize/Recognize.cs:72:    [Verb("listen", HelpText = "Detects speech in a microphone input stream.")]
./speech/api/Recognize/Recognize.cs:79:    [Verb("rec", HelpText = "Detects speech in an audio file. Supports other file formats.")]
./vision/api/Detect/Program.cs:29:    [Verb("labels", HelpText = "Detect labels.")]
./vision/api/Detect/Program.cs:32:    [Verb("safe-search", HelpText = "Detect safe-search.")]
./vision/api/Detect/Program.cs:35:    [Verb("properties", HelpText = "Detect properties.")]
./vision/api/Detect/Program.cs:38:    [Verb("faces", HelpText = "Detect faces.")]
./vision/api/Detect/Program.cs:41:    [Verb("landmarks", HelpText = "Detect landmarks.")]
./vision/api/Detect/Program.cs:44:    [Verb("text", HelpText = "Detect text.")]
./vision/api/Detect/Program.cs:47:    [Verb("logos", HelpText = "Detect logos.")]
./vision/api/Detect/Program.cs:50:    [Verb("crop-hint", HelpText = "Detect crop hint in a local image file.")]
./vision/api/Detect/Program.cs:53:    [Verb("web", HelpText = "Find web pages with matching images.")]
./vision/api/Detect/Program.cs:56:    [Verb("doc-text", HelpText = "Detect text in a document image.")]
./kms/api/Program.cs:31:    [Verb("encrypt", HelpText = "Encrypt the data in a text file using Cloud KMS.")]
./kms/api/Program.cs:34:    [Verb("decrypt", HelpText = "Decrypt the data in a text file using Cloud KMS.")]
./kms/api/Program.cs:58:    [Verb("listKeyRings", HelpText = "List all key rings associated with your project for a given location.")]
./kms/api/Program.cs:67:    [Verb("listCryptoKeys", HelpText = "List all crypto keys for the specified key ring.")]
./kms/api/Program.cs:70:    [Verb("createKeyRing", HelpText = "Create a key ring for holding crypto keys.")]
./kms/api/Program.cs:73:    [Verb("getKeyRing", HelpText = "Get a key ring's full path details and its create time.")]
./kms/api/Program.cs:86:    [Verb("createCryptoKey", HelpText = "Create a crypto key.")]
./kms/api/Program.cs:89:    [Verb("getCryptoKey", HelpText = "Get an crypto key's details.")]
./kms/api/Program.cs:104:    [Verb("disableCryptoKeyVersion", HelpText = "Disable a crypto key version.")]
./kms/api/Program.cs:107:    [Verb("enableCryptoKeyVersion", HelpText = "Enable a crypto key version.")]
./kms/api/Program.cs:110:    [Verb("getCryptoKeyVersion", HelpText = "Get a crypto key version.")]
./kms/api/Program.cs:113:    [Verb("destroyCryptoKeyVersion", HelpText = "Destroy a crypto key version.")]
./kms/api/Program.cs:116:    [Verb("restoreCryptoKeyVersion", HelpText = "Restore a crypto key version.")]
./kms/api/Program.cs:133:    [Verb("getCryptoKeyIamPolicy", HelpText = "Get a crypto key's IAM policy.")]
./kms/api/Program.cs:147:    [Verb("addMemberToCryptoKeyPolicy", HelpText = "Add member to crypto key's IAM policy.")]
./kms/api/Program.cs:164:    [Verb("removeMemberFromCryptoKeyPolicy", HelpText = "Remove member from crypto key's IAM policy.")]
./kms/api/Program.cs:181:    [Verb("getKeyRingIamPolicy", HelpText = "Get a keyRing's IAM policy.")]
./kms/api/Program.cs:192:    [Verb("addMemberToKeyRingPolicy", HelpText = "Add member to a keyRing's IAM policy.")]
rennie@rennie:~/gitrepos/dotnet-docs-samples$ 
```